packages/lesswrong/server/scripts/migrateCollections.ts

```diff
@@ -218,7 +225,7 @@ const copyData = async (
     }).count();
 
     if (totalCount < 1) {
-      return;
+      continue;
```

Comment:
Could you explain the change here? Was this a bug?

Reply:
Yeah, the bug here is that we don't have any AdvisorRequests, so it would try to copy data for it, and then skip copying data for all the other collections (while still creating tables for them and changing their read/write targets).